src/components/inputs/Search/SearchBarHome/SearchBarHome.tsx

```diff
@@ -0,0 +1,21 @@
+import { ReactComponent as SearchIcon } from '../../../icons/Search.svg';
+import './styles.css';
+import { searchBar } from '../../../../services/types';
+
+function SearchBarHome({ searchButtonAction }: searchBar) {
```

Comment:
```suggestion
function SearchBar({ searchButtonAction }: searchBar) {
```

we'll want to use the same search bar with some different effects in the nav bar, so let's give it a generic name